msteams-platform/apps-in-teams-meetings/meeting-app-extensibility.md

```diff
@@ -134,13 +134,13 @@ With post-meeting app experience, you can view the results of the meeting, such
 
 The following image displays the **Contoso** tab with results of poll and feedback received from meeting attendees:
 
-# [Desktop](#tab/desktop)
+# [Desktop](#tab/desktop3)
 
-:::image type="content" source="~/assets/images/apps-in-meetings/post.png" alt-text="Contoso tab with results.":::
+:::image type="content" source="~/assets/images/apps-in-meetings/post.png" alt-text="The screenshot is an example that shows the Contoso tab with results in desktop.":::
```

Comment:
To me the alt-text and the screenshot do not seem to be related or may be I am overlooking something.

Reply:
resolved